src/AccessLogEntry.h

```diff
@@ -61,6 +61,9 @@ class AccessLogEntry: public RefCountable
 
     void syncNotes(HttpRequest *request);
 
+    /// string representation of the reply headers
```

Comment:
Whatever that is, I do not think we are still using "string representation" in this context. If the packing code automatically masks some sensitive headers, then I suggest:

```suggestion
    /// dump all reply headers, masking sensitive headers for logging
```

If the packing code dumps everything as is:

```suggestion
    /// dump all reply headers (for sending or risky logging)
```

Adjust as needed, of course.

Reply:
Done.